### DISCUSSION TITLE
A bit different verification and use case

### DISCUSSION THREAD

**Discussion by dumblob** [uwu-tech/Kind#237] [General]
This is no bug. It's just a note to let potentially interested parties know, there are cool verification use cases in DeFi - see e.g. [Cairo](https://github.com/starkware-libs/cairo-lang ) (I'd recommend reading https://medium.com/starkware/hello-cairo-3cb43b13b209 ).

**Comment by rigille** [comment]
that seems interesting, I'll need to read more to understand it
